src/utils/work/dispatchWorkRequest.js

```diff
@@ -0,0 +1,55 @@
+import dayjs from 'dayjs';
+
+import fetchAPI from 'utils/http/fetchAPI';
+
+const getWorkerTimeout = (taskName, defaultTimeout) => {
```

Comment:
This was already around from before this PR, right? (if it was added in this PR I would suggest using `getTimeoutForWorkerTask` instead of this

Reply:
it was already there yes